packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -185,6 +185,7 @@ const Bulletin = ({
   const isAudio = mediaType === 'audio';
   const bulletinType = isAudio ? 'radio' : 'tv';
   const BulletinWrapper = isAudio ? RadioBulletinWrapper : TVBulletinWrapper;
+  const hiddenTextProps = lang ? { lang } : {};
```

Comment:
I think this will just add the `lang` prop as a boolean and we want `lang={lang}`.